src/payment/strategies/sage-pay/sage-pay-payment-strategy.ts

```diff
@@ -48,4 +70,9 @@ export default class SagePayPaymentStrategy extends CreditCardPaymentStrategy {
 
         return super.finalize(options);
     }
+
+    private _isThreeDSTwoExperimentOn(): boolean {
+        return this._store.getState().config.getStoreConfigOrThrow()
+        .checkoutSettings.features['INT-4994.Opayo_3DS2'] === true;
```

Comment:
Could we put this in one line? I believe your earlier approach is clear and tidy, though spanning across multiple lines. This approach is also good, but I'm concerned that the line break in between might confuse developers.